devices/common_patches/l0_syscfg_cfgr.yaml

```diff
@@ -35,6 +36,12 @@ SYSCFG:
       - ENBUF_BGAP_ADC
       - EN_BGAP
     _add:
+      # REF_LOCK is wrongly marked as writeable
+      REF_LOCK:
+        description: SYSCFG_CFGR3 lock bit
+        bitOffset: 31
+        bitWidth: 1
+        access: read-only
```

Comment:
Are you sure this is `read-only`? In RM0367 Rev 5 I see:

![image](https://user-images.githubusercontent.com/47219/68334783-38d7fd00-00d3-11ea-9807-3997874b7490.png)

And its access is marked as "rs" i.e. read/set.

Reply:
You're right, will fix tomorrow

Reply:
Okay, looks like the SVDs are still wrong by marking the bit write-only, so now they're patched to read-write.